source/reading.md

```diff
@@ -808,126 +784,97 @@ Unlike SQLite,
-import sqlalchemy
-from sqlalchemy import create_engine
-
-# connection_str = "postgresql://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>"
-connection_str = "postgresql://user0001:abc123@fakeserver.stat.ubc.ca:5432/can_mov_db"
-db = create_engine(connection_str)
-conn_mov_data = db.connect()
-
+```python
+conn = ibis.postgres.connect(database = "can_mov_db", 
+                                 host = "fakeserver.stat.ubc.ca",
+                                 port = 5432,
+                                 user = "user0001",
+                             password = "abc123"
+                            )
 ```
 
-After opening the connection, everything looks and behaves almost identically
-to when we were using an SQLite database in Python. For example, we can again use
-`table_names` to find out what tables are in the `can_mov_db` database:
+Aside from needing to provide that additional information, `ibis` makes it so
+that connecting to and working with a Postgres database is identical to
+connecting to and working with an SQLite database. For example, we can again use
+`list_tables` to find out what tables are in the `can_mov_db` database:
 
-```
-tables = conn_mov_data.table_names()
-tables
+```python
+conn.list_tables()
 ```
 
-
-```
+```text
 ['themes', 'medium', 'titles', 'title_aliases', 'forms', 'episodes', 'names', 'names_occupations', 'occupation', 'ratings']
-
 ```
 
 We see that there are 10 tables in this database. Let's first look at the
 `"ratings"` table to find the lowest rating that exists in the `can_mov_db`
-database. To access the table's contents we first need to declare the `metadata` of the table 
-and store it in a variable named `ratings`. Then, we can use the `select` function to 
-refer to the data in the table and return the result in python using `fetchall` function, just like 
-we did for the SQLite database.
-
-```
-metadata = MetaData(bind=None)
-ratings = Table(
-    'ratings', 
-    metadata, 
-    autoload=True, 
-    autoload_with=db
-)
-
-query = select([ratings])
-ratings_proxy = conn_mov_data.execute(query).fetchall()
-
-```
-
+database. 
 
+```python
+ratings_table = conn.table("ratings")
+ratings_table
 ```
-[('The Grand Seduction', 6.6, 150),
-('Rhymes for Young Ghouls', 6.3, 1685),
-('Mommy', 7.5, 1060),
-('Incendies', 6.1, 1101),
-('Bon Cop, Bad Cop', 7.0, 894),
-('Goon', 5.5, 1111),
-('Monsieur Lazhar', 5.6,610),
-('What if', 5.3, 1401),
-('The Barbarian Invations', 5.8, 99
-('Away from Her', 6.9, 2311)]
 
+```text
+AlchemyTable: ratings
+  title           string
+  average_rating  float64
+  num_votes       int64
 ```
 
-```{index} SQLAlchemy; select
+```{index} ibis; select
 ```
 
 To find the lowest rating that exists in the data base, we first need to
-extract the `average_rating` column using `select`:
+select the `average_rating` column:
 
-```
-avg_rating_db = select([ratings.columns.average_rating])
-avg_rating_db
+```python
+avg_rating = ratings_table[["average_rating"]]
```

Comment:
Are the double `[[` / `]]` needed here or is single enough?

Reply:
I'm sticking with double for consistency with Ch1, which only introduces selecting columns by putting a list between the sq bracks.